v0/destinations/blueshift/transform.js

```diff
@@ -158,20 +158,9 @@ const groupResponseBuilder = async (message, category, { Config }) => {
 
   const response = defaultRequestConfig();
 
-  const email = getFieldValueFromMessage(message, "email");
-  const id = getFieldValueFromMessage(message, "userIdOnly");
-  if (id) {
-    payload.identifier_key = "customer_id";
-    payload.identifier_value = id;
-  } else if (email) {
-    payload.identifier_key = "email";
-    payload.identifier_value = email;
-  } else {
-    throw new CustomError(
-      "[Blueshift] For group call customer_id or email is required to identify customer.",
-      400
-    );
-  }
+  // identifier_value mapped with userId, so identifier_key takes customer_id.
+  payload.identifier_key = "customer_id";
```

Comment:
Lets revert the updates, as there can be cases where some would want prefer email and choose not to send userId

Reply:
Lets add email if present along with id (as multi identifier key value pairs are supported)